engines/rbd.c

```diff
@@ -188,8 +197,26 @@ static int _fio_rbd_connect(struct thread_data *td)
 		log_err("rbd_open failed.\n");
 		goto failed_open;
 	}
+
+	/* add for rbd poll */
+    rbd->fd = eventfd(0, EFD_NONBLOCK);
+    if (rbd->fd < 0)
```

Comment:
Spaces again, and you are using mixed styles of braces. Please the starting { on the same line as the if statement.